test/Microsoft.IdentityModel.Protocols.Tests/HttpDocumentRetrieverTests.cs

```diff
@@ -91,6 +92,12 @@ public void GetMetadataTest(DocumentRetrieverTheoryData theoryData)
             {
                 aex.Handle((x) =>
                 {
+                    if (x.Data.Count > 0)
+                    {
+                        Assert.True(x.Data.Contains(HttpDocumentRetriever.StatusCode));
```

Comment:
We generally try to use CompareContext.AddDiff() rather than relying on Asserts, as Assert failures will cause the test to terminate and will not notify us of failures further on in the test.

Reply:
use context.adddiff in latest commit